### ISSUE TITLE
Don't let this extension die

### ISSUE THREAD

**Issue by DaAwesomeP** [YePpHa/YouTubeCenter#2077]
There is been a significant drop of support, improvements, and bugfixes since the beginning of the year. YouTube Center barely works anymore. However, there are no other good alternatives. @YePpHa I would recommend finding some contributors to help manage the repository. Don't pick me, but I'm sure that there are some people that have contributed a lot already. Giving a few people access to commit, merge pull requests, and close issues will enormously help. The extension barely works (I have many many bugs) and it needs a lot of work.

Thanks,
DaAwesomeP

**Comment by Eisys** [comment]
[Youtube+](https://github.com/ParticleCore/Particle/)

**Comment by TacoDelPaco** [comment]
Thanks @Eisys!

**Comment by LiamBM** [comment]
@Eisys It's a shame Youtube+ doesn't support Safari. As someone who uses Safari, YoutubeCentre is my only choice left.

**Comment by Eisys** [comment]
People've said the Safari version is so old and broken..
#ChromeMasterRace

**Comment by LiamBM** [comment]
The YoutubeCentre Safari version doesn't work at all anymore. They've completely stopped trying to fix it. Chrome is getting pretty slow on macs, so I prefer not switching back to it.

**Comment by Eisys** [comment]
@LiamBM Perhaps you could try [Tampermonkey](http://tampermonkey.net/?browser=safari) for Safari and then use Youtube+ as a userscript.

**Comment by LiamBM** [comment]
@Eisys yeah, I've tried that already, however with the user script some of the features like "auto pause" and "auto resolution" don't work. Pretty annoying. I'm using YoutubeCentre with Tampermonkey at the moment and most of the features that I use still work. I'm just hoping that YoutubeCentre doesn't die out soon.

**Comment by Lycerion** [comment]
The dev version is still working great for me. But yes, absolutely get some help!

**Comment by vpontin** [comment]
@Yonezpt take this

**Comment by Yonezpt** [comment]
This is not a project I would be capable to take in the current time even if YePpHa would want it, my schedule is currently very full and I would have to rework YTC massively to address the reported memory leaks as well as to understand YTC's inner workings.

To be honest it would probably be easier to write YTC from scratch, 27 000 lines of code are not easy to debug.

**Comment by IMNdi** [comment]
Why is this not a Patreon thing or a Kickstarter?

This is one of the best addons out there, and requires maintenance constantly. Are you sure there's nobody out there that would take this task upon themselves if there's something to be gained?

Surely YTC could be trimmed down somewhat, for example dropping glow, simplifying player sizes, and (depending how hard it is) dropping Flash. With Mozilla finally fixing their autoplay in 41 we don't need the Flash any longer so they can be stopped. Plus, H5 players are easier to control.

IMO, the biggest issue with YTC is that it tries to do everything and that is costly from every angle. There are good addons out there, no need to duplicate everything.

Breaking the addon into several different addons would keep things light, make it easier to find leaks and issues and even let the user use some features when other features are broken.

YTC has 60K users and it's broken. I imagine it'd have more if it worked, surely you have issues working on a monolith - but if it were more modular you could let other people pitch in since it would be easier to understand.

**Comment by Zarggg** [comment]
As has been stated many times before, the issue with updating YTC is not financial; it is entirely dependent on the time and interest of the author.

At this point, the only options are to wait until @YePpHa is able to work on it again or for an intrepid coder to fork the project and maintain it themselves.

This discussion has happened many times in the past and the same things keep being said. Repeating these topics serves nothing except to clutter up the issue tracker.

**Comment by IMNdi** [comment]
I wasn't suggesting an YTC update.

**Comment by DaAwesomeP** [comment]
The purpose of this issue is not to identify the problem or try to figure out why. It is to solve it. I'm calling on anyone with enough time, dedication, and understanding of the extension to help maintain it or fork it. I don't have said time or understanding, so I'm trying to find someone else or at least until @YePpHa can do something or assign someone. The last commit is from August 16th.

**Comment by komocode** [comment]
Should we set up some bounties? https://www.bountysource.com

**Comment by DaAwesomeP** [comment]
@komocode If you're willing to pay, then sure! By looking at the Pulse graph, someone is closing issues, so someone who is active can accept pulls *if they have access right*. However, only @YePpHa really can make releases.
